build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Maybe change to build-image, or something to specify that this is for creating an image

Reply:
Sure, I can do that.